pkg/cri/client/v1alpha2/client.go

```diff
@@ -0,0 +1,901 @@
+		return nil, err
+	}
+
+	resp, err := c.rsc.PodSandboxStats(ctx, &req)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := alphaRespToV1(resp, v1resp); err != nil {
+		return nil, err
+	}
+
+	return v1resp, nil
+}
+
+func (c *client) ListPodSandboxStats(ctx context.Context, in *v1.ListPodSandboxStatsRequest, opts ...grpc.CallOption) (*v1.ListPodSandboxStatsResponse, error) {
+	if err := c.checkRuntimeService(); err != nil {
+		return nil, err
+	}
+
+	var (
+		req    api.ListPodSandboxStatsRequest
+		v1resp = &v1.ListPodSandboxStatsResponse{}
+	)
+
+	if err := v1ReqToAlpha(in, &req); err != nil {
+		return nil, err
+	}
+
+	resp, err := c.rsc.ListPodSandboxStats(ctx, &req)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := alphaRespToV1(resp, v1resp); err != nil {
+		return nil, err
+	}
+
+	return v1resp, nil
+}
+
+func (c *client) UpdateRuntimeConfig(ctx context.Context, in *v1.UpdateRuntimeConfigRequest, opts ...grpc.CallOption) (*v1.UpdateRuntimeConfigResponse, error) {
+	if err := c.checkRuntimeService(); err != nil {
+		return nil, err
+	}
+
+	var (
+		req    api.UpdateRuntimeConfigRequest
+		v1resp = &v1.UpdateRuntimeConfigResponse{}
+	)
+
+	if err := v1ReqToAlpha(in, &req); err != nil {
+		return nil, err
+	}
+
+	resp, err := c.rsc.UpdateRuntimeConfig(ctx, &req)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := alphaRespToV1(resp, v1resp); err != nil {
+		return nil, err
+	}
+
+	return v1resp, nil
+}
+
+func (c *client) Status(ctx context.Context, in *v1.StatusRequest, opts ...grpc.CallOption) (*v1.StatusResponse, error) {
+	if err := c.checkRuntimeService(); err != nil {
+		return nil, err
+	}
+
+	var (
+		req    api.StatusRequest
+		v1resp = &v1.StatusResponse{}
+	)
+
+	if err := v1ReqToAlpha(in, &req); err != nil {
+		return nil, err
+	}
+
+	resp, err := c.rsc.Status(ctx, &req)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := alphaRespToV1(resp, v1resp); err != nil {
+		return nil, err
+	}
+
+	return v1resp, nil
+}
+
+/*
+func (c *client) CheckpointContainer(ctx context.Context, in *v1.CheckpointContainerRequest, opts ...grpc.CallOption) (*v1.CheckpointContainerResponse, error) {
+	return nil, fmt.Errorf("unimplemented by CRI v1alpha2 RuntimeService")
+}
+
+func (c *client) GetContainerEvents(ctx context.Context, in *v1.GetContainerEventsRequest, opts ...grpc.CallOption) (v1.RuntimeService_GetContainerEventsClient, error) {
+	return nil, fmt.Errorf("unimplemented by CRI v1alpha2 RuntimeService")
+}
+
+func (c *client) ListMetricDescriptors(ctx context.Context, in *v1.ListMetricDescriptorsRequest, opts ...grpc.CallOption) (*v1.ListMetricDescriptorsResponse, error) {
+	return nil, fmt.Errorf("unimplemented by CRI v1alpha2 RuntimeService")
+}
+
+func (c *client) ListPodSandboxMetrics(ctx context.Context, in *v1.ListPodSandboxMetricsRequest, opts ...grpc.CallOption) (*v1.ListPodSandboxMetricsResponse, error) {
+	return nil, fmt.Errorf("unimplemented by CRI v1alpha2 RuntimeService")
+}
+*/
```

Comment:
Why commented?

Reply:
They are being defined but not available yet in the CRI API version we use.